project/cookiecutter.json

```
@@ -112,6 +116,11 @@
       "__prompt__": "Add GitHub Action to Deploy this project?",
       "1": "Yes",
       "0": "No"
+    },
+    "devops_gitlab_deploy": {
+      "__prompt__": "Add GitLab Action to Deploy this project?",
```

I found no term "GitLab Action". It appears they call it the inelegant, yet accurate, mouthful "GitLab CI/CD". Also lower case "deploy". Please replace all instances of both.

```suggestion
      "__prompt__": "Add GitLab CI/CD to deploy this project?",
```